modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/update/SyncUpdateResponseParser.kt

```diff
@@ -91,7 +91,11 @@ class SyncUpdateResponseParser(
             "UserFolder" -> readFolder(reader, response)
             "UserPodcast" -> readPodcast(reader, response)
             "UserEpisode" -> readEpisode(reader, response)
-            "UserBookmark" -> readBookmark(reader, response)
+            "UserBookmark" -> if (featureFlagWrapper.isEnabled(Feature.BOOKMARKS_ENABLED)) {
+                readBookmark(reader, response)
+            } else {
+                reader.skipValue()
+            }
```

Comment:
I would be inclined to include keep the feature check if-else logic in the `readBookmark` method myself like it was before. I don't think we'd ever want to read the bookmark if the feature was disabled, and that would keep the different branches of this when statement more consistent. Definitely not a big deal though. Moving it here is fine if you prefer.